src/Cake.Issues/IIssueComparer.cs

```diff
@@ -157,5 +186,17 @@ private static int GetHashCode(params object[] objects)
                 return hash;
             }
         }
+
+        private static int GetDictionaryHashCode(IReadOnlyDictionary<string, string> dictionary)
```

Comment:
There are some StyleCop violations here:

- SA1204: Static members should appear before non-static members
- SA1507: Code should not contain multiple blank lines in a row